langchain/chat_models/azure_openai.py

```diff
@@ -110,3 +110,15 @@ def _default_params(self) -> Dict[str, Any]:
             **super()._default_params,
             "engine": self.deployment_name,
         }
+
+    def _create_chat_result(self, response: Mapping[str, Any]) -> ChatResult:
+        generations = []
+        for res in response["choices"]:
+            if "finish_reason" in res and res["finish_reason"] == "content_filter":
+                raise ValueError("Azure has not provided the response due to a content"
+                                 " filter being triggered")
+            message = _convert_dict_to_message(res["message"])
+            gen = ChatGeneration(message=message)
+            generations.append(gen)
+        llm_output = {"token_usage": response["usage"], "model_name": self.model_name}
+        return ChatResult(generations=generations, llm_output=llm_output)
```

Comment:
For the same reason as above you need to import ChatResult

Reply:
Imported 👍